src/components/productSection/productSection.jsx

```diff
@@ -0,0 +1,26 @@
+import style from './productSection.module.css';
+
+export const ProductSection = ({ title, description, image, reverse }) => {
+  const [firstWord, ...restWords] = title.split(' ');
+  const formattedTitle = (
+    <>
+      <span className={style.highlighted}>{firstWord}</span>{' '}
+      {restWords.join(' ')}
+    </>
+  );
```

Comment:
Create a completely separate component instead in a separate file in the `ProductSection` directory.
```
export const FormattedTitle = () => {
  const [firstWord, ...restWords] = title.split(' ');
  return (
    <h4>
      <span className={style.highlighted}>{firstWord}</span>{' '}
      {restWords.join(' ')}
    </h4>
  )
}
```